crawler/file_processing.py

```diff
@@ -284,10 +289,18 @@ def __init__(self, file_name: str, centre: Centre):
             FIELD_DATE_TESTED,
         }
 
-        # These are to allow some variability in headers,
-        # due to receiving inconsistent file formats
+        # These are to allow some variability in headers, due to receiving inconsistent file formats
         self.header_regex_correction_dict = {r"^Root Sample$": FIELD_ROOT_SAMPLE_ID}
 
+    def is_consolidated(self) -> bool:
```

Comment:
This doesn't feel right to me. I think the type of the file should be a propery of `CentreFile` not `Centre`. We're also performing the match twice, once to find all the files for the centre and then whenever we call these methods from the CentreFile instance. I think we could have an Enum which is passed in when creating the CentreFile which describes the type.

Reply:
Ok, yes, makes sense, a centre file should know after being created what type of centre it is. Also the condition checking could be ```centre_file.surveillance_type == 'eagle' (or 'unconsollidated, 'genomic_consollidated')```

Reply:
Do you mean "a centre file should know after being created what type of ~~centre~~ file it is.."?

Reply:
Happy to chat more about this. Do you want to include this change or make it later?

Reply:
TODO before UAT